cldfbench_doreco.py

```diff
@@ -220,12 +220,10 @@ def cmd_makecldf(self, args):
                 "dc:bibliographicCitation": CORPUS_CITATION_FMT.format(**row)
             })
 
-            print(row["Language"], row["Family"])
             args.writer.objects["LanguageTable"].append({
                 "ID": row["Glottocode"],
                 "Name": row["Language"],
                 "Glottocode": row["Glottocode"],
-                "Family": row["Family"],
```

Comment:
We got a funny regression here. Now we add "Family" to the schema - but not to the data :) Will fix.